docs/developers/add-new-analytics-module.md

```diff
@@ -0,0 +1,19 @@
+# Analytics module 
+
+This document describes how to add a new analytics module to Prebid Server. 
+
+The PBS Analytics creates _loggable objects_ for a transaction at each endpoint (for example, an `AuctionObject` for the `/openrtb2/auction` endpoint, `CookieSyncObject` for the  `/cookiesync` endpoint, `SetUIDObject` for the `/setuid` endpoint, and an `AmpObject` for the `/amp` endpoint.)
```

Comment:
Is this relevant to someone who just wants to add an analytics module?

Reply:
you're right, i'lll update it